sips/sip-122.md

```diff
@@ -25,38 +25,47 @@ This SIP proposes several improvements to the sUSD shorts  [SIP 103](https://sip
+
+2) The use of the underlying in the computation of the collateral price, provides added protection against the risk of actors taking advantage of periods of low liquidity by pushing up the price of the ERC-20 equivalent far above it's peg, in order to drawdown  more synth from the contract than is permitted.
 
-Using the minimum price function instead of just using the underlying price directly mitigates against both these attack vectors and protects the debt pool from these tail risks. However, it is important to mention that this improvement does not protect against the risk of the external collateral value deviating far away from it's peg on outstanding loans. Therefore, it is important for the Spartan Council to consider how well these ERC-20 based representation of the underlying tracks the price of the underlying.
+Using the minimum price function instead of just using the underlying price directly mitigates against both these attack vectors and protects the debt pool from these tail risks. However, it is important to mention that this improvement does not protect against the risk of the external collateral value deviating far away from it's peg on outstanding loans. Therefore, it is important for the Spartan Council to consider how well these ERC-20  representation of the underlying tracks the price of the underlying.
 
 
-### Interaction Circuit Breakers on Shorts:
+### Interaction Circuit Breakers on Shorts and Borrows:
 
-The circuit-breakers outlined in [SIP 55](https://sips.synthetix.io/sips/sip-55) and [SIP 65](https://sips.synthetix.io/sips/sip-65) protect the debt pool from the risk of significant price changes that could result in irrecoverable loss of value to minters. However, the current shorting contracts do not have this level of protection baked-in. This improvement proposes to disable certain interactions when the shorted currency is not tradeable on the exchange. The interactions that would be disabled are those that involve drawing out liquidity from the contract or changing the value of the loan. Therefore, collateral deposits would still be permitted, but repayment, liquidation, drawdown and collateral withdrawals would all be disabled.
+The circuit-breakers outlined in [SIP 55](https://sips.synthetix.io/sips/sip-55) and [SIP 65](https://sips.synthetix.io/sips/sip-65) protect the debt pool from the risk of significant price changes that could result in irrecoverable loss of value to minters. However, the current shorting contracts do not have this level of protection baked-in. This improvement proposes to disable certain interactions when the shorted currency or borrowed synth is not tradeable on the exchange. The interactions that would be disabled are those that involve drawing out liquidity from the contract or changing the value of the loan. Therefore, collateral deposits would still be permitted, but repayment, liquidation, drawdown and collateral withdrawals would all be disabled.
```

Comment:
Borrowing synths should still work here after the upgrade ie,  ETH -> sUSD, renBTC -> sBTC even in the scenario that the market is closed for sBTC?

Reply:
well if the circuit breaker took place on btc, than any synth borrowing might be done at a bad price 🤔  so in my mind just disable everything except deposits with the circuit breaker until the smoke clears. Not sure this is clear or not?

Reply:
That makes sense , to also prevent borrowing the asset if the price feed is invalid / circuit breaker is on.